.github/workflows/cd-internal-testnet.yml

```diff
@@ -1,21 +1,40 @@
 name: Continuous Deployment (Internal Testnet)
 # run after every successful CI job of new commits to the master branch
+# if deploy version or config has changed
 on:
   workflow_run:
     workflows: [Continuous Integration (Kava Master)]
     types:
       - completed
 
 jobs:
+  changed_files:
+    runs-on: ubuntu-latest
+    # define output for first job forwarding output of changedInternalTestnetConfig job
+    outputs:
+        changedInternalTestnetConfig: ${{ steps.changed-internal-testnet-config.outputs.any_changed }}
+    steps:
+        - uses: actions/checkout@v4
+          with:
+            fetch-depth: 0 # OR "2" -> To retrieve the preceding commit.
+        - name: Get all changed internal testnet files
+          id: changed-internal-testnet-config
+          uses: tj-actions/changed-files@v42
+          with:
+            # Avoid using single or double quotes for multiline patterns
+            files: |
+              ci/env/kava-internal-testnet/**
```

Comment:
might be an issue if internal testnet version is set to `MASTER`. But might be okay? Since protonet can be used if latest testnet is needed.

Reply:
not following, if they set the version to master, that will count as a changed file causing internal testnet to deploy. If they want to deploy a different version, they will have to change the version again causing a change to be detected and internal testnet deployed.

Reply:
what I mean is change to MASTER -> then internal testnet will be deployed as expected. Then on future master merges, the change file here will indicate no version or genesis changes, and thus not redeploy on master changes.